src/main/resources/org/jenkinsci/plugins/durabletask/heartbeat-launcher.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
```

Comment:
So that's what i thought at first, but i came across something in a google groups mentioning that syscall is actually being frozen and now new changes are being done to it and that's it's poor documentation.  But i'll actually go look for the comment to see if it's an officially unofficial thing.  For what it's worth, in the code, the syscall functions do pointo the to x/sys/ARCH code.